packages/amplify-cli-core/src/__tests__/customPoliciesUtils.test.ts

```diff
@@ -136,4 +136,15 @@ describe('Custom policies util test', () => {
 
     expect(template.Resources?.CustomLambdaExecutionPolicy).toBeUndefined();
   });
+  test('test generateCustomPoliciesInTemplate with empty action array', () => {
+    (stateManager.getCustomPolicies as jest.Mock).mockReturnValueOnce([
+      {
+        Action: undefined,
+        Resource: [],
+      },
+    ]);
+    expect(() => generateCustomPoliciesInTemplate({}, 'lambdaResourceName', 'Lambda', 'function')).toThrowError(
+      /Invalid custom IAM policies for .*?/,
```

Comment:
Can we also assert the `Type` of the `Error` as well, e.g. `CustomPoliciesFormatError` or `AmplifyError`.